build-env.sh

```diff
@@ -84,7 +85,7 @@ echo "Dynatrace PAAS Token   : $DT_PAAS_TOKEN"
 
 home_folder="/home/$shell_user"
 
-echo "$shell_user:$shell_user" | chpasswd
+# echo "$shell_user:$shell_user" | chpasswd
```

Comment:
this can't be removed from GCP as this sets the password for the shell user that can only authenticate with an ssh key, this can however be moved under [this](https://github.com/dynatrace-ace/perform2021-vhot-monaco/blob/master/acebox-gcp.tf#L90) line